### ISSUE TITLE
The result when setting L_s as non-zero?

### ISSUE THREAD

**Issue by yangxh11** [chou141253/FGVC-PIM#10]
I have notice that in your paper, the L_s is set as 0. It is consistent with the code:
``
def _select_loss(self, selected_logits, labels):
        loss1 = 0
``

But I am curious about the experiment result of a non-zero L_s. Does the accuracy decrease evidently？

**Comment by chou141253** [comment]
In our experiments, using GCN combiner and L_s at the same time may cause unstable training. We would take more experiments on this, thanks.
